guides/release/getting-started/core-concepts.md

```diff
@@ -74,10 +74,10 @@ In the following example, the [`didRender()`](https://emberjs.com/api/ember/rele
 ```javascript {data-filename=/app/components/foo-did-render-example.js}
 import Component from '@ember/component';
 
-export default Component.extend({
+export default class FooDidRenderExample extends Component {
```

Comment:
only glimmer components should have examples in native class syntax

Reply:
This example should be updated to `@glimmer/component`